upup/pkg/fi/cloudup/networking.go

```diff
@@ -57,3 +58,20 @@ func usesCNI(c *api.Cluster) bool {
 	glog.Warningf("Unknown networking mode configured")
 	return true
 }
+
+// TODO: we really need to sort this out:
+// https://github.com/kubernetes/kops/issues/724
+// https://github.com/kubernetes/kops/issues/626
+// https://github.com/kubernetes/kubernetes/issues/30338
+const defaultCNIAsset = "https://storage.googleapis.com/kubernetes-release/network-plugins/cni-07a8a28637e97b22eb8dfe710eeae1344f69d16e.tar.gz"
```

Comment:
I think this is fine, but most go people seem to use this style
```
const (
  a = 1
  b = 2
)
```

etc.

I don't particularly mind though!